src/testing/flow_solver_cases/flow_solver_case_base.h

```diff
@@ -34,6 +34,9 @@ class FlowSolverCaseBase
     /// Pure Virtual function to generate the grid
     virtual std::shared_ptr<Triangulation> generate_grid() const = 0;
 
+    /// Set higher order grid
```

Comment:
Seems like the default implementation is an empty function? Did you mean to set is a pure virtual function (=0)? Then it seems like the only derived class that uses that function is the naca0012 case.

Should this only be defined in the naca0012 case?

Reply:
If it were a pure virtual function, all flow solver cases would have to implement it, which is not necessarily the case as not all cases have a higher order grid. 

The other tests currently only use the generate_grid function, which is pure virtual. The problem with using that function in this case is that adding the higher order grid requires having access to dg, which generate_grid does not. So I can either modify generate_grid to take a dg (and then comment out /*dg*/ in all tests that don't require a dg to set the grid). Or I can make a new function that always gets called by FlowSolver, and is set to do nothing in FlowSolverCaseBase, but if needed can be overrided with the appropriate behaviour.